.github/workflows/main.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         run: make tidy
       - name: Build
         run: make mpi-operator.v2
+      - name: generate codes
+        run: make verify-generate
```

Comment:
As discussed in https://github.com/kubeflow/mpi-operator/pull/509#pullrequestreview-1271884158, we verify generated codes.